website/docs/zephyr-training/01_What_is_Zephyr/west.md

```diff
@@ -70,11 +70,11 @@ commands will still be `west build` and `west flash`.
 :::note
 
 We will not be using `west flash` during this training because the Codespaces
-container that serve as the build environment don't have access to the USB port
-on your computer.
+container that serves as the development environment does not have access to the
```

Comment:
Updated "build environment" to "development environment" because we're using Codespaces for more than just building the firmware (has an editor, etc)